internal/services/user_device_service.go

```diff
@@ -101,7 +101,7 @@ func (uds *userDeviceService) CreateUserDevice(ctx context.Context, deviceDefID,
 	ud := models.UserDevice{
 		ID:                 userDeviceID,
 		UserID:             userID,
-		DeviceDefinitionID: dd.DeviceDefinitionId,
+		DeviceDefinitionID: dd.NameSlug,
```

Comment:
the legacy `DeviceDefinitionID` should still be the ksuid legacy id. It would be the new column that would get set. Realize `DefinitionID` is confusing for the new name, but the idea is to remove the DeviceDefinitionID eventually so...